.gitignore

```
@@ -3,7 +3,7 @@ target/
 !.mvn/wrapper/maven-wrapper.jar
 !**/src/main/**/target/
 !**/src/test/**/target/
-
+!**/node_modules
```

this would add your node modules to git 
[gitignore exceptions](https://riptutorial.com/git/example/911/exceptions-in-a--gitignore-file)